imagetest/test_suites/security/image_security_test.go

```diff
@@ -295,9 +295,7 @@ func verifyAutomaticUpdate(image string) error {
 		if interval > maxInterval || interval < minInterval {
 			return fmt.Errorf("autoclean interval is invalid or an unexpected length")
 		}
-	default:
-		return fmt.Errorf("unsupported image %s", image)
-	}
+  }
```

Comment:
indent

Reply:
there's no reason to give style feedback in golang; gofmt will apply a consistent style.